assets/src/scripts/monitoring-location/components/hydrograph/selectors/axes.js

```diff
@@ -77,7 +78,7 @@ export const getBrushXAxis = createSelector(
     getBrushXScale('current'),
     getTsTimeZone,
     getCurrentDateRangeKind,
-    (xScale, ianaTimeZone, period) => createXAxis(xScale, period, ianaTimeZone)
+    (xScale, ianaTimeZone, period) => createXAxis(xScale, ianaTimeZone, period)
```

Comment:
Looks like we only need the time zone and not the period.

Reply:
Oh, yup, duh. I'll fix it!